ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/DataStorageConfiguration.java

```diff
@@ -69,14 +69,16 @@ default Unstable getUnstable() {
   @Value.Immutable
   interface Unstable {
 
-    boolean DEFAULT_BONSAI_LIMIT_TRIE_LOGS_ENABLED = false;
+    boolean DEFAULT_BONSAI_LIMIT_TRIE_LOGS_ENABLED = true;
     long MINIMUM_BONSAI_TRIE_LOG_RETENTION_LIMIT = DEFAULT_BONSAI_MAX_LAYERS_TO_LOAD;
     int DEFAULT_BONSAI_TRIE_LOG_PRUNING_WINDOW_SIZE = 30_000;
     boolean DEFAULT_BONSAI_FULL_FLAT_DB_ENABLED = true;
     boolean DEFAULT_BONSAI_CODE_USING_CODE_HASH_ENABLED = true;
 
     DataStorageConfiguration.Unstable DEFAULT =
-        ImmutableDataStorageConfiguration.Unstable.builder().build();
+        ImmutableDataStorageConfiguration.Unstable.builder()
+            .bonsaiLimitTrieLogsEnabled(true)
```

Comment:
this should be redundant since the default is true now, right?

Reply:
removed here [`100bf43` (#7181)](https://github.com/hyperledger/besu/pull/7181/commits/100bf43dd97419a42f8e92bc7390ec651e466ea5)